packages/eslint-plugin/src/rules/naming-convention.ts

```diff
@@ -154,19 +154,24 @@ const PREFIX_SUFFIX_SCHEMA: JSONSchema.JSONSchema4 = {
     type: 'string',
     minLength: 1,
   },
-  minItems: 1,
   additionalItems: false,
 };
 type JSONSchemaProperties = Record<string, JSONSchema.JSONSchema4>;
 const FORMAT_OPTIONS_PROPERTIES: JSONSchemaProperties = {
   format: {
-    type: 'array',
-    items: {
-      type: 'string',
-      enum: util.getEnumNames(PredefinedFormats),
-    },
-    minItems: 1,
-    additionalItems: false,
+    oneOf: [
+      {
+        type: 'array',
+        items: {
+          type: 'string',
+          enum: util.getEnumNames(PredefinedFormats),
+        },
+        additionalItems: false,
```

Comment:
how it should behave whe you provide `[]` empty array? as right now you removed `minItems: 1,`

Reply:
`[] === null`
I didn't document it because I'd rather people be explicit, but people assumed it worked this way anyways.